src/main/java/seedu/address/model/person/Name.java

```diff
@@ -12,7 +11,7 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
-    /*
+    /**
```

Comment:
This is a multi-line comment rather than a JavaDoc. Perhaps we can make it into a JavaDoc by adding the param and return but otherwise we can keep it as a comment using a single asterisk